src/func/Microsoft.Health.Fhir.Ingest.Host/IomtConnectorFunctions.cs

```diff
@@ -57,7 +57,9 @@ public static class IomtConnectorFunctions
                 EnsureArg.IsNotNull(templateDefinitions, nameof(templateDefinitions));
                 EnsureArg.IsNotNull(events, nameof(events));
 
-                var template = CollectionContentTemplateFactory.Default.Create(templateDefinitions);
+                var templateContext = CollectionContentTemplateFactory.Default.Create(templateDefinitions);
+                templateContext.EnsureValid();

```

Comment:
Suggestion.  Can shorten these two lines if the EnsureValid returns the template, i.e. var template = templateContext.Template.EnsureValid();  Seems a little cleaner.

Reply:
Discussed offline, I'll check on this in the next revision.